_opensearch/install/rpm.md

```diff
@@ -5,154 +5,472 @@ parent: Install OpenSearch
 ---
 
-# RPM
-
-The RPM Package Manager (RPM) installation provides everything you need to run OpenSearch inside Red Hat or Red Hat-based Linux Distributions.  
+{% comment %}
+The following liquid syntax declares a variable, major_version_mask, which is transformed into "N.x" where "N" is the major version number. This is required for proper versioning references to the Yum repo.
+{% endcomment %}
+{% assign version_parts = site.opensearch_major_minor_version | split: "." %}
 
-RPM supports CentOS 7 and 8, and Amazon Linux 2. If you have your own Java installation and set `JAVA_HOME` in your terminal application, macOS works, as well.
+# RPM
 
-There are two methods for installing OpenSearch on RPM: 
+Installing OpenSearch using RPM Package Manager (RPM) simplifies the process considerably compared to the [Tarball]({{site.url}}{{site.baseurl}}/opensearch/install/tar/) method. Several technical considerations, such as the installation path, location of configuration files, and the creation of a service managed by `systemd`, as examples, are handled automatically by the package manager.
 
-## Manual method
+Generally speaking, installing OpenSearch from the RPM distribution can be broken down into a few steps:
 
+1. **Download and install OpenSearch.**
+   - Installation is performed by manually installing from an RPM distribution, or by creating a local YUM repository and installing with `yum`.
+1. **Configure important system settings.**
+   - These settings are applied to the host before modifying any OpenSearch files.
+1. **(Optional) Test OpenSearch.**
+   - Confirm that OpenSearch is able to run before you apply any custom configuration.
+   - This can be done without any security (no password, no certificates) or with a demo security configuration that can be applied by a packaged script.
+1. **Configure OpenSearch for your environment.**
+   -  Apply basic settings to OpenSearch and start using it in your environment.
 
-1. Download the RPM package directly from the [OpenSearch downloads page](https://opensearch.org/downloads.html){:target='\_blank'}. The RPM package can be download both as `x64` and `arm64`. 
+The RPM distribution provides everything you need to run OpenSearch inside Red Hat or Red Hat-based Linux Distributions, such as supported CentOS and RHEL versions, and Amazon Linux 2. If you have your own Java installation and set `JAVA_HOME` in your terminal application, macOS works, as well.
 
-2. Import the public GPG key. This key verifies that the your OpenSearch instance is signed.
+This guide assumes that you are comfortable working from the Linux command line interface (CLI). You should understand how to input commands, navigate between directories, and edit text files. Some example commands reference the `vi` text editor, but you may use any text editor available.
+{:.note}
 
-   ```bash
-   sudo rpm --import https://artifacts.opensearch.org/publickeys/opensearch.pgp
-   ```
+## Download and install OpenSearch
```

Comment:
Should this be "Step 1:"? "Step 2: Configure important system settings" seems to be missing an older sibling.

Reply:
Yes, it should Thank you!